pytrip/util.py

```diff
@@ -43,3 +45,35 @@ def f(x):
 
     f.__name__ = name
     return f
+
+
+def volume_histogram(cube, voi=None, bins=256):
+    """
+    Generic volume histogram calculator, useful for DVH and LVH or similar.
+
+    :params cube: a data cube of any shape, e.g. Dos.cube
+    :params voi: optional voi where histogramming will happen.
+    :returns [x],[y]: coordinates ready for plotting. Dose (or LET) along x, Normalized volume along y in %.
+
+    If VOI is not given, it will calculate the histogram for the entire dose cube.
+
+    Providing voi will slow down this function a lot, so if in a loop, it is recommended to do masking
+    i.e. only provide Dos.cube[mask] instead.
+    """
+
+    if voi is None:
+        mask = None
+    else:
+        vcube = voi.get_voi_cube()
+        mask = (vcube.cube == 1000)
+
+    _xrange = (0.0, cube.max()*1.1)
+    _hist, x = np.histogram(cube[mask], bins=bins, range=_xrange)
+    _fhist = np.flip(_hist, axis=0)  # reverse historgram, so first element is for highest dose
+    _fhist = np.cumsum(_fhist)
+    _hist = np.flip(_fhist, axis=0)  # flip back again to normal representation
```

Comment:
For python older than 3.4 we need to use numpy < 1.12, see:
https://github.com/pytrip/pytrip/blob/master/requirements.txt

Older version of numpy do not have `flip` method.
Consider using sth like:
```
x = x[::-1]
```

This is the reason travis job fails: https://travis-ci.org/pytrip/pytrip/jobs/226449935